src/event-listeners/onLayoutChanged.js

```diff
@@ -8,8 +8,8 @@ export default dispatch => (e, { added, removed }) => {
     dispatch(actions.setTotalPages(totalPages));
 
     setTimeout(() => {
-      // this 'onLayoutChange' handler get trigger before the 'onLayoutChange' handler on 'viewerPageManager' get fire (so page number aren't updated yet)
-      // use 'setTimeout' so the annotationManager.onLayoutChange handler finish before we call 'setPageLabels'
+      // this 'onLayoutChange' handler get trigger before the 'onLayoutChange' handler on 'viewerPageManager' get trigger (so page numbers aren't updated yet)
+      // use 'setTimeout' so the other 'onLayoutChange' handler finishs before we call 'setPageLabels'
```

Comment:
I think it should read: this 'onLayoutChange' handler get(s) trigger(ed).

It should be annotManager instead of 'viewerPageManager' I think.

finishs -> finishes?

Reply:
I updated the comment